roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -125,6 +125,10 @@ locals {
     zenith_fqdn_monitoring = "{{ zenith_fqdn_monitoring }}"
     zenith_sshd_host = "{{ zenith_sshd_host }}"
     zenith_sshd_port = "{{ zenith_sshd_port }}"
+    azimuth_workstation_username = "{{ cluster_user_username | default('azimuth') }}"
+    azimuth_workstation_uid = "{{ cluster_user_uid | default('1005') }}"
+    azimuth_workstation_gid = "{{ cluster_user_gid | default('1005') }}"
+    azimuth_workstation_is_sudo = "{{ cluster_user_is_sudo | default(true) }}"
```

Comment:
The names of these metadata items will need to be changed to the generic names when the changes requested on the images patch are made.